src/lib/hardware/pci.lua

```diff
@@ -110,6 +113,24 @@ function set_bus_master (device, enable)
    C.close(fd)
 end
 
+--- ### Open a device
```

Comment:
Do we really want to reintroduce `open_device()`? (I don't see it being used in this change set?)